### PR TITLE
[gollum] add guide for gollum

### PR DESCRIPTION
**Pull request by pReya**
Gollum is a simple wiki software written in Ruby. It is the software component that drives many popular wiki integrations (e.g. GitHub and GitLab wikis). It is a very simple, small and fast wiki alternative to larger systems like Dokuwiki. Gollum is Git-backed, so all changes made through the web interface are also committed to the underlying Git repository.

### CODE REVIEW

File: source/guide_gollum.rst [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+  [isabell@stardust ~]$ uberspace web backend set / --http --port 4567
+  Set backend for / to port 4567; please make sure something is listening!
+  You can always check the status of your backend using "uberspace web backend list".
+  [isabell@stardust ~]$
+
+.. note:: If you want to run Gollum under a different URL, just adjust the "/" parameter in the command above to match your preferred path.
+
+
+Now you can start Gollum again and you should be able to open it in your browser using your Uberspace URL.
```
Comment (EV21) [inline source/guide_gollum.rst:116]: To make the guides more maintainable there are prepared snippets.
```suggestion
.. note:: By default Gollum runs on port 4567, so we need to create a web backend for this port. 
.. include:: includes/web-backend.rst

.. note:: If you want to run Gollum under a different URL, just adjust the "/" parameter in the command above to match your preferred path.

Now you can start Gollum again and you should be able to open it in your browser using your Uberspace URL (e.g. ``https://isabell.uber.space/``).
```

the snippet for `.. include:: includes/web-backend.rst` currently is:
```
To make the application accessible from the outside, configure a `web backend <webbackend_>`_:

::

  [isabell@stardust ~]$ uberspace web backend set / --http --port <port>
  Set backend for / to port <port>; please make sure something is listening!
  You can always check the status of your backend using "uberspace web backend list".
  [isabell@stardust ~]$

.. _webbackend: https://manual.uberspace.de/web-backends.html

```

File: source/guide_gollum.rst [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+::
+
+  [isabell@stardust ~]$ supervisorctl reread
+  gollum: available
+  [isabell@stardust ~]$ supervisorctl update
+  gollum: added process group
+  [isabell@stardust ~]$ supervisorctl start gollum
+  gollum: started
+  [isabell@stardust ~]$
```
Comment (EV21) [inline source/guide_gollum.rst:140]: To make the guides more maintainable there are prepared snippets.
```suggestion
.. include:: includes/supervisord.rst
```
`supervisorctl update` also starts the service, so it makes more sense to check the status.  
`supervisorctl stop`and `supervisorctl start` makes sense (e.g.) when you perform updates to your application.
the snippet for `.. include:: includes/supervisord.rst` currently is:
```
After creating the configuration, tell :manual:`supervisord <daemons-supervisord>` to refresh its configuration and start the service:

.. code-block:: console

 [isabell@stardust ~]$ supervisorctl reread
 SERVICE: available
 [isabell@stardust ~]$ supervisorctl update
 SERVICE: added process group
 [isabell@stardust ~]$ supervisorctl status
 SERVICE                            RUNNING   pid 26020, uptime 0:03:14
 [isabell@stardust ~]$


```

**Comment by pReya** [comment]
@EV21 Thanks for the feedback! Will work through it as soon as possible (hopefully on the upcoming weekend).

**Comment by luto** [comment]
I agree with the feedback @EV21 gave :)

on hold until the guide has been adapted.

**Comment by pReya** [comment]
@luto Mind checking again? Merged all suggestions from @EV21

File: source/guide_gollum.rst [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+
+Install recent CMake version
+----------------------------
+
+To install a recent CMake version, the fastest way is to run the official installer script in your home folder (alternatively you could also use brew to install the current CMake version). Make sure to copy the link to the latest version from the `official website <https://cmake.org/download/>`_ (look for "Latest Release" and ``Linux x86_64``).
+
+::
+
+  [isabell@stardust ~]$ bash <(wget -qO- https://github.com/Kitware/CMake/releases/download/v3.17.3/cmake-3.17.3-Linux-x86_64.sh)
```
Comment (luto) [inline source/guide_gollum.rst:40]: This doesn't seem to work :/ can you try again in a fresh uberspace?

```
Do you accept the license? [yn]: 
y
By default the CMake will be installed in:
  "/home/gollum2/cmake-3.17.3-Linux-x86_64"
Do you want to include the subdirectory cmake-3.17.3-Linux-x86_64?
Saying no will install in: "/home/gollum2" [Yn]: 
y

Using target directory: /home/gollum2/cmake-3.17.3-Linux-x86_64
Extracting, please wait...

[gollum2@belinda ~]$ ll /home/gollum2/cmake-3.17.3-Linux-x86_64
total 0
```

Comment (pReya) [reply source/guide_gollum.rst]: Fixed it.

**Comment by luto** [comment]
@pReya I adapted the guide a bit, feel free to have a look :) thank you for you guide!

**Comment by pReya** [comment]
@luto They all look like very good additions. Thanks for the work! Glad to have it merged.

**Pull request merged**